aten/src/ATen/NamedTensorUtils.cpp

```diff
@@ -461,20 +453,20 @@ optional<std::vector<Dimname>> compute_cat_outnames(TensorList tensors) {
   return result;
 }
 
-optional<std::vector<Dimname>> compute_matmul_outnames(
+std::vector<Dimname> compute_matmul_outnames(
```

Comment:
I like this change. I think I mentioned before I generally prefer avoiding optional/nullable lists wherever possible.

Reply:
I like the type safety that optional gives but I got too many headaches dealing with `optional<ArrayRef>` and `optional<vector>` so I decided to kill it here.